sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -6042,7 +6042,7 @@ pi_result _pi_queue::synchronize() {
     // Otherwise sync all L0 queues/immediate command-lists.
     for (auto &QueueMap : {ComputeQueueGroupsByTID, CopyQueueGroupsByTID}) {
       for (auto &QueueGroup : QueueMap) {
-        if (Device->ImmCommandListUsed) {
+        if (UsingImmCmdLists) {
```

Comment:
why is this change needed?

Reply:
Everywhere else we simply use the Queue's value UsingImmCmdLists. This was the one instance where an additional indirection was done. The code is equivalent. I changed it for consistency.